src/interfaces/network.py

```diff
@@ -63,17 +63,17 @@ def perform_test(self):
         # Combination of protocol and test
         test_results = {}
         for proto in self.proto_suites:
-            # start the protocol
-            proto.start(self)
```

Comment:
```python
if not proto.is_distributed():
  proto_config = proto.get_config()
  proto_config.hosts = [test.config.client_host, test.config.server_host]
```
add is_distributed to IProtoSuite, default return True; but for CSProtocol, it return False.